### PR TITLE
Adding agent-nginx for Open DC/OS

### PR DESCRIPTION
**Pull request by malnick**

### CODE REVIEW

**Comment by malnick** [comment]
@spahl This is ready for review.

**Comment by spahl** [comment]
Please rebase and fix conflicts.

File: nginx.agent.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    include common/http.conf;
+
+    server {
+        server_name agent.mesos;
+        include common/server.conf;
+
+        listen 61001 default_server;
+
+        location /system/health/v1 {
```
Comment (branden) [inline nginx.agent.conf]: The definition for `/system/health/v1` is the same for both `nginx.master.conf` and `nginx.agent.conf` (except for a JWT auth step that gets no-opped due to disabled security anyway.) So this location can be DRY'ed out of the individual conf files and into `common/server.conf`.

**Comment by malnick** [comment]
@spahl Rebased.

**Comment by malnick** [comment]
@spahl I need to re-test this against dcos/dcos before we merge.

**Comment by malnick** [comment]
Updated https://github.com/dcos/dcos/pull/292

**Pull request merged**